packages/kit/src/runtime/client/utils.js

```diff
@@ -136,7 +136,7 @@ export function get_link_info(a, base, uses_hash_router) {
 
 	const external =
 		!url ||
-		!!target ||
+		target === '_blank' ||
```

Comment:
should it be this instead? We can't know for certain if the browsing context will be the same as ours for `_top` etc
```suggestion
		(target  || '_self') !== '_self' ||
```

Reply:
Would `location.ancestorOrigins` work to check if there are no parents/ancestors? https://developer.mozilla.org/en-US/docs/Web/API/Location/ancestorOrigins

If there is a way to check for parent browsing contexts then the page wouldn't need to reload when these values default to `_self`.

> _parent: The parent browsing context of the current one. If no parent, behaves as _self.
>
> _top: The topmost browsing context. To be specific, this means the "highest" context that's an ancestor of the current one. If no ancestors, behaves as _self.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target